docs/building-your-quest/writing-quest-texts/index.md

```diff
@@ -8,6 +8,14 @@ nav_order: 10
 
 # Writing Quest Texts
 
+## Quest Descriptions
+
+Every quest needs a title, a tagline, and a quest description.
+
+- **The title** can be 2-80 characters. Try to aim for 5-30 characters, as the name is a part of the quest cover image, and you don't want to use a tiny font that occupies the whole image. Puns are welcome, but please don't make them lewd or otherwise inappropriate. Feel free to browse our current [quest catalog](https://www.trywilco.com/catalog) for inspiration.
+- **The tagline** can be 2-1000 characters. Try to aim for 5-6 words. Should be a narrative-driven, short sentence. Fun, geeky references and puns are acceptable, but you can also go for something more descriptive.
```

Comment:
Isn't 1000 characters a bit too much? 😅

Reply:
Yeah - if we want it to be 5-6 words, seems like 1000 is too much